### PR TITLE
Fixed white space and placement style problems.

### PR DESCRIPTION
**Pull request by OliviaRS**

### CODE REVIEW

File: src/Commands/AutonomousCommand.h [hunk 1]
```diff
@@ -23,10 +11,7 @@
  */
```
Comment (MajorDarius) [inline src/Commands/AutonomousCommand.h]: This comment doesn't have a partner, do we need it?

Comment (MajorDarius) [inline src/Commands/AutonomousCommand.h]: Do we need this comment? If not we should get rid of it.

File: src/Commands/TankDrive.h [hunk 2]
```diff
@@ -20,11 +10,8 @@
  */
```
Comment (MajorDarius) [inline src/Commands/TankDrive.h]: Same thing as the other one, do we need this comment?

**Review by MajorDarius — CHANGES_REQUESTED** [review]
Check out comments but everything else looks good

File: src/Commands/AutonomousCommand.h (hunk 1 above)
Comment (duane701) [reply src/Commands/AutonomousCommand.h]: @MajorDarius, are you referring to the end of comment shown in the diff view or the entire comment shown in the file?

Remember that the diff view does not show the entire file. Only the changes to the file are shown in the diff view. 

However, looking at the file, I'm not sure that the comment actually has any value. But we don't need to change it for this PR.

File: src/OI.cpp [hunk 3]
```diff
@@ -65,8 +65,10 @@ OI::OI()
 {
   // Process operator interface input here.
 
+/*    We'll need these later but keep this like this for now.
   static const double kRPMNudge = 10.0;
-  static const double kPositionNudge = 0.;
+  static const double kPositionNudge = 0;
```
Comment (duane701) [inline src/OI.cpp]: Double and float should always include a decimal point. This is technically assigning an integer to a double. The integer is promoted to a double. If the number includes the decimal point, then a double is assigned to a double, not conversion is required.

File: src/Robot.cpp [hunk 4]
```diff
@@ -85,31 +85,41 @@ void Robot::TeleopPeriodic() {
...
+  SmartDashboard::PutBoolean("Gear RevBanner sensor true=triggered", GearPickup::getInstance()->IsRevLimitSwitchClosed());
+  SmartDashboard::PutBoolean("Gear Pneumatic false=low", GearPickup::getInstance()->IsGearUp());
+  SmartDashboard::PutBoolean("Gear Squeezer false=open", GearPickup::getInstance()->IsSqueezeOpen());
+  SmartDashboard::PutNumber("Gear Roller EncoderRPM", GearPickup::getInstance()->GetGearIntakeRPM());
+
+  //...............BallConveyor SmartDashboard.....
+  SmartDashboard::PutNumber("Ball Conveyor EncoderRPM", BallConveyor::getInstance()->GetBallConveyorRPM());
+
+  //...............Lifter SmartDashboard.......
+  SmartDashboard::PutNumber("Lifter OutPut Voltage", Lifter::getInstance()->GetCurrent());
```
Comment (duane701) [inline src/Robot.cpp]: "OutPut" should be "Output", and this appears to be mislabeled. The function is called `GetCurrent()`.

**Review by duane701 — CHANGES_REQUESTED** [review]
FTP

**Review by MajorDarius — APPROVED** [review]
LGTM

**Comment by duane701** [comment]
Outdated. Redone in a later PR.

**Pull request closed without merging**